### ISSUE TITLE
Update documentation regarding SQLite support

### ISSUE THREAD

**Issue by RayBB** [wallabag/wallabag#7779]
I ran into a bit of confusion about the status of SQLite and thought it would be good to update the docs to reflect the current situation.

The [FAQ](https://doc.wallabag.org/en/user/faq.html#why-has-sqlite-support-been-dropped) says:
> Since June 2017, [SQLite support has been dropped](https://github.com/wallabag/wallabag/pull/3171) and MySQL is the default database now.

However, this doesn't seem to be accurate. SQLite is still the default and according to [this](https://github.com/wallabag/wallabag/issues/2766#issuecomment-490918115) 2019 comment support is still around.

**Comment by nicosomb** [comment]
Hello @RayBB ! 

As you can see in the parameters file https://github.com/wallabag/wallabag/blob/2.6/app/config/parameters.yml.dist#L13, we define mysql as the default database.

**Comment by RayBB** [comment]
@nicosomb I missed that as I have been using the docker setup, which uses `pdo_sqlite` as the default database. 

In any case, I think the line about SQLite support being dropped should probably be removed to avoid confusion.

**Comment by nicosomb** [comment]
No, these lines are necessary: they mean that we can't propose support for sqlite installation.

**Comment by RayBB** [comment]
Thank you for clarifying.
If it's not supported at all then why is SQLite the default in the [official docker image](https://github.com/wallabag/docker), which is the [suggested](https://doc.wallabag.org/en/developer/docker.html) way to run wallabag in production?

**Comment by nicosomb** [comment]
Maybe a mistake to fix 😊

**Comment by eMUQI** [comment]
> Why has SQLite support been dropped?
> Since June 2017, [SQLite support has been dropped](https://github.com/wallabag/wallabag/pull/3171) and MySQL is the default database now. SQLite support has been dropped because it's awful for database migrations. It's just you'll have to do migrations the hard way and lose data each time.
> For development purposes, SQLite is still fine though.

In fact, I just migrated my wallabag instance from my Raspberry Pi 4 to Pi 5. I used Docker Compose for deployment and opted for SQLite. The migration experience was excellent; all I needed to do was copy the original database file to the new machine and set the correct read/write permissions (this is very important—initially, I didn't do this, which prevented wallabag from writing to the database, and it took me a while to troubleshoot).

This leaves me a bit puzzled because my actual experience doesn't align with what the documentation suggests. SQLite has been very reliable during my migration process. In fact, as long as you know the steps for migration, the whole process is quite straightforward. The reason it took me some time was that I didn't know the steps initially.

Additionally, I chose to use SQLite because the [wallabag/docker](https://github.com/wallabag/docker?tab=readme-ov-file#sqlite) mentions: 

> The easiest way to start wallabag is to use the SQLite backend.

 And I haven't encountered any issues so far. Can anyone explain why the official documentation states that SQLite is not suitable for migrations?

**Comment by eMUQI** [comment]
> No, these lines are necessary: they mean that we can't propose support for sqlite installation.

@nicosomb I lack technical knowledge about databases, but as a user, I find SQLite reliable and easy to use. It requires no complex configuration and seems lightweight. Will wallabag keep this option available?

**Comment by nicosomb** [comment]
Today, we won't really plan to remove totally SQLite (we use it for development for example). But writing migrations for SQLite is a pain for us ...

**Comment by chevdor** [comment]
I have been testing quite a bunch of self hosted service and the question postegress/mysql vs sqlite comes often.
I am getting better perf out of the box using sqlite for small databases. It is much easier to manage and backup.
There are old threads about sqlite being unreliable over old NFS shares for instance but this is no longer the case with newer NFS. Moreover, backups are so fast and simple that one can litterally backup every hour.
All of that to 👍 maintaining sqlite support. I am currently using postgres and even plan to go back to sqlite.
